libgadget/blackhole.c

```diff
@@ -47,12 +46,11 @@ typedef struct {
     MyFloat BH_MinPotPos[3];
     MyFloat BH_MinPot;
 
-    int BH_TimeBinLimit;
+    short int BH_minTimeBin;
     MyFloat FeedbackWeightSum;
 
     MyFloat Rho;
     MyFloat SmoothedEntropy;
-    MyFloat SmoothedPressure;
```

Comment:
I recall pressure was used to testing if pressure makes more sense with pressure-entropy. The verdict was it didn't matter. So yes for removal.

Reply:
Ok!